### ISSUE TITLE
Can I eject the SD Card after? 

### ISSUE THREAD

**Issue by tomasvelososantos** [jilleb/mib2-toolbox#278]
Can I eject the SD card after all the procedures?

**Comment by jilleb** [comment]
Yes, no problem. But .. if you want to apply some changes, keep an SD card in because it's needed for backups and such
